codebuild/ci/validate-ci.yml

```
@@ -0,0 +1,55 @@
+version: 0.2
+
+env:
+  variables:
+    REGION: us-east-1
+    DOMAIN: crypto-tools-internal
+    REPOSITORY: java-esdk-ci
+  parameter-store:
+    ACCOUNT: /CodeBuild/AccountId
+
+phases:
+  install:
+    commands:
+      - pip install awscli
+    runtime-versions:
+      java: $JAVA_ENV_VERSION
+  pre_build:
+    commands:
+      - VERSION_HASH="$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)-$CODEBUILD_RESOLVED_SOURCE_VERSION"
+      - export SETTINGS_FILE=$(pwd)/codebuild/ci/settings.xml
+      - git clone https://github.com/aws-samples/busy-engineers-document-bucket.git
+      - cd busy-engineers-document-bucket/exercises/java/encryption-context-complete
+      - export CODEARTIFACT_TOKEN=$(aws codeartifact get-authorization-token --domain $DOMAIN --domain-owner $ACCOUNT --query authorizationToken --output text --region ${REGION})
+      - export CODEARTIFACT_REPO_URL=https://${DOMAIN}-${ACCOUNT}.d.codeartifact.${REGION}.amazonaws.com/maven/${REPOSITORY}
+  build:
+    commands:
+      - |
+        mvn verify \
+          -Pcodeartifact \
+          -Dcheckstyle.skip \
+          -Desdk.version=$VERSION_HASH \
+          -Dmaven.compiler.target=$JAVA_NUMERIC_VERSION \
+          -Dmaven.compiler.source=$JAVA_NUMERIC_VERSION \
+          -Dcodeartifact.token=$CODEARTIFACT_TOKEN \
+          -Dcodeartifact.url=$CODEARTIFACT_REPO_URL \
+          --no-transfer-progress \
+          -s $SETTINGS_FILE
+  post_build:
+    commands:
+      - cp -r $CODEBUILD_SRC_DIR/src/test/ src/test
+      - cp -r $CODEBUILD_SRC_DIR/src/examples/java/com src/main/java/
+      - cp $CODEBUILD_SRC_DIR/codebuild/ci/busy-bucket-pom.xml pom.xml
+      - |
+        mvn test \
```

```suggestion
        mvn verify \
```

Trying to figure out why this worked on my local but not in the codebuild project...